modules/util/portal-tools-db-partition-virtual-instance-migration-extractor/README.markdown

```diff
@@ -0,0 +1,18 @@
+# DB Partition Virtual Instance Migration Extractor Tool
```

Comment:
This applies to both README:
- Provide always a way to use a single letter. PartitionId should be schemaName. With that you can use -u for user, -j jbdcUrl, -p password, -s schemaName, -d output directory
- Avoid the word "Set the" since does not add much. Just describe the parameter purpose.
- Use sh scripts for the execution
- user and pass are mixed